src/test/kotlin/domain/map/MineMapTest.kt

```diff
@@ -62,4 +63,76 @@ class MineMapTest : BehaviorSpec({
             }
         }
     }
+
+    Given("지뢰 개수 8 초기화 속성이 준비 되었을 떄") {
+        val mineSweeperInitProperty = MineSweeperInitProperty(
+            height = 100.toPositive(),
+            width = 50.toPositive(),
+            mineCount = 8.toPositive()
+        )
+        When("맵을 만들면") {
+            val mockMineCoordinates = setOf(
+                Coordinate(1, 1),
+                Coordinate(2, 1),
+                Coordinate(3, 1),
+                Coordinate(1, 2),
+                Coordinate(3, 2),
+                Coordinate(1, 3),
+                Coordinate(2, 3),
+                Coordinate(3, 3),
+            )
+
+            val mineMapFactory = RealMineMapFactory(MockMineCoordinatesCreator { mockMineCoordinates })
```

Comment:
`MineCoordinatesCreator`를 [SAM](https://kotlinlang.org/docs/fun-interfaces.html)으로 변경하신다면, 아래와 같이 람다식으로도 변경해볼 수 있어요! 😃

```suggestion
            val mineMapFactory = RealMineMapFactory { mockMineCoordinates }
```